Libraries/test/TestPowerShellFunctions/PowerShellScriptsAsFunctions/aws-lambda-tools-defaults.json

```diff
@@ -11,8 +11,7 @@
   "profile":"default",
   "region" : "us-west-1",
   "configuration" : "Release",
-  "framework" : "netcoreapp2.1",
```

Comment:
The framework isn't needed anymore or should this be netcoreapp3.1?

Reply:
In the old versions of the tooling we had to have it set. Now a days the tooling will infer the framework from whatever the project is targeting.